EntAppProject/src/test/java/com/jobook/EntAppProject/UserServiceUnitTests.java

```diff
@@ -21,62 +21,41 @@
 public class UserServiceUnitTests {
 	@Autowired
 	IUserService userService;
-	
-	User user;
-	
-	@Before
-	public void BeforeEachTest()
-	{
-		user = null;
-	}
-	
+
 	@Test
 	public void customerLogin()
 	{
-		customerUserTriesToLogin();
-		
+		User user = userService.login("test", "Password");
 		assertTrue(user instanceof Customer);
 		assertNotNull(user);
 	}
-	
-	private void customerUserTriesToLogin() {
-		user = userService.login("test", "Password");
-	}
 
 	@Test
 	public void employeeLogin()
 	{
-		employeeUserTriesToLogin();
+		User user = userService.login("testEmployee", "Password");
 		assertTrue(user instanceof Employee);
 		assertNotNull(user);
 	}
-	
-	private void employeeUserTriesToLogin() {
-		user = userService.login("testEmployee", "Password");
-	}
 
 	@Test
 	public void badLogin()
 	{
-		badUserTriesToLogin();

```

Comment:
Same comment as above, this is really hard to read, and it directly mutates the state.